android/src/main/AndroidManifest.xml

```diff
@@ -200,7 +200,7 @@
         </receiver>
         <receiver
             android:name="com.thebluealliance.androidclient.gcm.GCMBroadcastReceiver"
-            android:permission="com.google.android.c2dm.permission.SEND">
+            android:permission="${gcmPermissionRequired}">
```

Comment:
Can you just comment that this is defined in build.gradle so it's easy to find in the future?